libraries/adsk/adsklib/colorcorrect.mtlx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<materialx version="1.38" xmlns:xi="http://www.w3.org/2001/XInclude">
+<materialx version="1.37" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
We keep all current files 1.37 until 1.38 is released -- allowing for the upgrade path to occur by default.